src/lib/draw/point_tool.ts

```diff
@@ -10,12 +10,17 @@ import { colors, circleRadius } from "../../colors";
 
 const source = "edit-point-mode";
 
+// Properties are guaranteed to exist
+type FeatureWithProps<G extends Geometry> = Feature<G> & {
+  properties: { [name: string]: any };
```

Comment:
I'm still confused why we need the extra `{}` here to express that it's a map from string to any, but... this is what type-checks